extensions/websockets-next/pom.xml

```diff
@@ -12,6 +12,7 @@
 
     <artifactId>quarkus-websockets-next-parent</artifactId>
     <name>Quarkus - WebSockets Next</name>
+    <description>A new implementation of the WebSocket API that is more efficient and easier to use. It is also designed to work efficiently with Quarkus' reactive programming model and the Quarkus' networking layer.</description>
```

Comment:
Where does the description on code.quarkus.io come from?

Reply:
Update: Thinking about it for two seconds, a description being in the runtime pom.xml should be fine. So I'm confused about what's going on for this extension and https://quarkus.io/extensions/io.quarkus/quarkus-confluent-registry-avro/. 

I've confirmed at https://registry.quarkus.io/q/swagger-ui/#/Client/get_client_extensions_all shows the registry is definitely missing the description. That isn't surprising, really, because it's what we read, but I'm now puzzled about why the registry data is incomplete, and also where the code.quarkus information is coming from. Will investigate more.

Reply:
Thanks for spotting the real problem, @gsmet!